src/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -330,7 +330,7 @@ export function getUiSettingDefaults() {
     },
     'savedObjects:perPage': {
       name: 'Objects per page',
-      value: 5,
+      value: 20,
```

Comment:
Hey @bmcconaghy / @chrisronline. Assume this is OK to change. Rest of this PR is visual, but 5 seemed like a super small default for a table.

Reply:
Yea sounds good to me